slidechain/cmd/peg/peg.go

```diff
@@ -130,7 +128,8 @@ func main() {
 
 // doPrePegIn calls the pre-peg-in Slidechain RPC.
 // That RPC builds, submits, and waits for the pre-peg TxVM transaction and records the peg-in in the database.
-func doPrePegIn(bcid, assetXDR []byte, amount, expMS int64, pubkey ed25519.PublicKey, slidechaind string) ([]byte, error) {
+func doPrePegIn(bcid, assetXDR []byte, amount, expMS int64, pubkey ed25519.PublicKey, slidechaind string) ([32]byte, error) {
+	var zeroBytes [32]byte
```

Comment:
This is awkward. A better solution is to declare `nonceHash` here and just return it wherever you're using `zeroBytes`.